src/main/java/seedu/eduke8/Parser.java

```diff
@@ -33,15 +47,17 @@ public void parseCommand(String userInput) {
         }
     }
 
-    public void parseAnswer(String quizAnswer) {
-        switch (quizAnswer.trim()) {
-        case "1":
-        case "2":
-        case "3":
-        case "4":
-        default:
-            break;
+    public void parseChosenOption(UiInterface ui, ArrayList<OptionInterface> options, QuestionInterface question) {
+        // Should probably use parser for this part to add hints also
+        int chosen = Integer.parseInt(ui.getInputFromUser());
```

Comment:
Would "chosenOption" be a better way to name it?

Reply:
Good point! Will make the recommended alteration right away.